chainlibpy/amino/__init__.py

```diff
@@ -1,25 +0,0 @@
-from .basic import (DEFAULT_BECH32_HRP_BASE, Coin, CommissionRates, Content,
-                    Description, Input, Output, StdFee, SyncMode,
-                    TimeoutHeight, VoteOptionAbstain, VoteOptionNo,
-                    VoteOptionNoWithVeto, VoteOptionUnspecified, VoteOptionYes)
-from .signdoc import StdSignDoc
-
-__all__ = [
-    "message",
-    "DEFAULT_BECH32_HRP_BASE",
-    "Coin",
-    "TimeoutHeight",
-    "CommissionRates",
-    "Content",
-    "Description",
-    "Input",
-    "Output",
-    "StdFee",
-    "StdSignDoc",
-    "SyncMode",
-    "VoteOptionAbstain",
-    "VoteOptionNo",
-    "VoteOptionNoWithVeto",
-    "VoteOptionUnspecified",
-    "VoteOptionYes",
-]
```

Comment:
I think we should keep the amino protocol, because later it will be used in the multisign method.

Reply:
In the new commit I pushed, I recovered all files under `chainlibpy/amino` directory. Thank you for pointing this out :)